Windows/MainWindow.xaml.cs

```diff
@@ -223,6 +247,74 @@ public partial class MainWindow {
                 panel.UpdateLayout();
             }
         }
+        private void ExportJson(string oFile = null)
+        {
```

Comment:
Please put this on the same line as everything else. I don't want mixed styles in here.

Reply:
Not sure what happened here...